DataFormats/PixelCPEFastParams/src/PixelCPEFastParams.cc

```diff
@@ -0,0 +1,9 @@
+#include "FWCore/Utilities/interface/typelookup.h"
+#include "DataFormats/PixelCPEFastParams/interface/PixelCPEFastParams.h"
+#include "DataFormats/TrackerCommon/interface/SimplePixelTopology.h"
+
+using PixelCPEFastParamsPhase1 = pixelCPEforDevice::PixelCPEFastParams<alpaka_common::DevHost, pixelTopology::Phase1>;
+using PixelCPEFastParamsPhase2 = pixelCPEforDevice::PixelCPEFastParams<alpaka_common::DevHost, pixelTopology::Phase2>;
+
+TYPELOOKUP_DATA_REG(PixelCPEFastParamsPhase1);
+TYPELOOKUP_DATA_REG(PixelCPEFastParamsPhase2);
```

Comment:
Do these EventSetup data products really need to be in `DataFormats`? `DataFormats` is primarily for Event data products (although I see two EventSetup data products currently defined in `DataFormats`, IIRC both have some dependence-related rationale)